### ISSUE TITLE
Code Indentation

### ISSUE THREAD

**Issue by griffinlacek** [browniecaker/prj-rev-bwfs-dasmoto#3]
Indentation is one of those not technically required, but really nice to have parts of coding. Your HTML indentation could use a bit more structure by indenting items from one another. For example, in HTML children elements are indented from their parents normally. This means all elements inside body should be indented from body. And all elements inside a div should be indented from that div. It's not code breaking, but it helps page flow and emphasizes that these elements are children of the parent element.

Basically, look at all elements that are wrapped by a parent element and indent those. And remember if two elements are wrapped by the same parent item then they should be indented equally!

Also, in CSS your indentation is pretty solid, but try to make sure all styles are aligned evenly. It makes code much cleaner and readable!
